src/monitoring_service/cli.py

```diff
@@ -53,6 +56,30 @@ def setup_logging(log_level: str) -> None:
     )
 
 
+def check_gas_reserve(web3: Web3, private_key: str) -> bool:
+    """ Check periodically for gas reserve in the account """
+    gas_price = web3.eth.gasPrice
+    gas_limit = GAS_REQUIRED_FOR_MS_MONITOR
+    estimated_required_balance = gas_limit * gas_price * 10
+    estimated_required_balance_eth = Web3.fromWei(estimated_required_balance, 'ether')
+    current_balance = web3.getBalance(private_key_to_address(private_key))
+    if current_balance < estimated_required_balance:
+        log.info('Missing gas reserve', required_wei=estimated_required_balance)
```

Comment:
it says wei here, but you converted it to eth before.

Reply:
Right, thanks. In Raiden, they log the WEI value but the show the ETH value to the user. Should we change or follow their approach?

Reply:
Ah, you're right. I missed that. Never mind then.